cdc/processor/manager.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/pingcap/tiflow/cdc/model"
 	"github.com/pingcap/tiflow/pkg/config"
 	cdcContext "github.com/pingcap/tiflow/pkg/context"
+	cerror "github.com/pingcap/tiflow/pkg/errors"
```

Comment:
```suggestion
  "github.com/pingcap/tiflow/pkg/errors"
```

Reply:
Ref: https://github.com/pingcap/tiflow/blob/master/pkg/errors/reexport.go

Reply:
we imported "github.com/pingcap/errors" in this file too, must rename the package here